zaza/openstack/charm_tests/rabbitmq_server/tests.py

```diff
@@ -39,7 +39,7 @@ class RmqTests(test_utils.OpenStackBaseTest):
     @classmethod
     def setUpClass(cls):
         """Run class setup for running tests."""
-        super(RmqTests, cls).setUpClass()
+        super(RmqTests, cls).setUpClass(application_name='rabbitmq-server')
```

Comment:
As this line is being modified, it's probably useful to do some code gardening and switch it to Python3 syntax:

```python
super().setUpClas(...)
```